diff --git a/node_modules/spotlight.js/src/js/spotlight.js b/node_modules/spotlight.js/src/js/spotlight.js
index fb7a8ff..45e4d13 100644
--- a/node_modules/spotlight.js/src/js/spotlight.js
+++ b/node_modules/spotlight.js/src/js/spotlight.js
@@ -105,9 +105,9 @@ let prefix_request, prefix_exit;
 
 addListener(document, "click", dispatch);
 
-export function init(){
+export function init({ force = false }){
 
-    if(body){
+    if(body && !force){
 
         return;
     }
